lib/ui/views/farm_claim/layouts/components/farm_claim_in_progress_popup.dart

```diff
@@ -83,14 +83,14 @@ class FarmClaimInProgressPopup {
     BuildContext context,
     WidgetRef ref,
   ) {
-    final farmClaim = ref.watch(FarmClaimFormProvider.farmClaimForm);
+    final farmClaim = ref.watch(farmClaimFormNotifierProvider);
     return aedappfm.PopupCloseButton(
       warningCloseWarning: farmClaim.isProcessInProgress,
       warningCloseLabel: farmClaim.isProcessInProgress == true
           ? AppLocalizations.of(context)!.farmClaimProcessInterruptionWarning
           : '',
       warningCloseFunction: () {
-        final _farmClaim = ref.read(FarmClaimFormProvider.farmClaimForm);
+        final _farmClaim = ref.read(farmClaimFormNotifierProvider);
         ref
           ..invalidate(
             DexFarmProviders.getFarmList,
```

Comment:
Warning about invalidate strategy. Farmlock features (deposit, level up, withdraw, claim) impact just Farmlock . not Farm legacy